lib/active_hash/base.rb

```diff
@@ -218,13 +218,20 @@ def field(field_name, options = {})
         field_names << field_name
 
         add_default_value(field_name, options[:default]) if options[:default]
-        define_getter_method(field_name, options[:default])
+        define_getter_method(field_name, options)
```

Comment:
think this is one last remnant of the previous refactoring

Reply:
I have to keep this if i want the `field :stuff, :private => true` syntax, don't I?